### PR TITLE
Bumps to v.1.8.0

### PR DESCRIPTION
**Pull request by gruyaume**

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -3,13 +3,12 @@ summary: Magma Orchestrator Controller
...
   control plane for various types of gateways in Magma.
-version: "1.6.1"
+version: "1.8.0"
 license: Apache-2.0
 base: ubuntu:22.04
 platforms:
   amd64:
-
-cmd: ["orchestrator", "-run_echo_server=true", "-logtostderr=true", "-v=0"]
+cmd: ["/usr/bin/envdir", "/var/opt/magma/envdir", "/var/opt/magma/bin/orchestrator", "-run_echo_server=true", "-logtostderr=true", "-v=0"]
```
Comment (gruyaume) [inline rockcraft.yaml]: Why would we want to use envdir,

File: rockcraft.yaml [hunk 2]
```diff
@@ -25,16 +24,23 @@ parts:
     plugin: go
     source: https://github.com/magma/magma/
     source-type: git
-    source-tag: v1.6.1
+    source-tag: v1.8.0
     source-subdir: orc8r/cloud/go
     build-snaps:
       - go
+    stage-packages:
+      - daemontools
```
Comment (gruyaume) [inline rockcraft.yaml]: Why use daemontools?

**Review by gruyaume — COMMENTED** [review]
Sorry for not catching it earlier, I don't see why we would want to use daemontools/envdir. I know they use it in the upstream project but with pebble I don't know why we would need it.

File: rockcraft.yaml (hunk 1 above)
Comment (saltiyazan) [reply rockcraft.yaml]: ✅

File: rockcraft.yaml (hunk 2 above)
Comment (saltiyazan) [reply rockcraft.yaml]: ✅

**Review by Gmerold — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Pull request merged**